### ISSUE TITLE
README not copied when building the library

### ISSUE THREAD

**Issue by caroso1222** [caroso1222/ng-lib-schematics#11]
The `README.md` must be copied over to `/dist`, otherwise the npm registry page will show an empty description. This is bad as the package won't properly appear in searches.

**Comment by NetanelBasal** [comment]
what about https://github.com/caroso1222/ng-lib-schematics/blob/master/gulpfile.js#L22?

**Comment by caroso1222** [comment]
Good catch, not sure why it's not properly working then, need to debug it thoroughly. By any chance, did it work for you when running `npm run build:lib`? Maybe you wanted to get rid of the readme file on purpose and I missed it?

**Comment by NetanelBasal** [comment]
Actually not, I checked it and it does not copy the README file.

**Comment by NetanelBasal** [comment]
The first link was a mistake, it's the schematics library gulpfile, not the library that uses it. We should check https://github.com/caroso1222/ng-lib-schematics/blob/master/src/lib-standalone/files/lib/__path__/build-tools/gulpfile.js#L72

**Comment by NetanelBasal** [comment]
I think I know what the problem is. You are coping the whole lib files but the README file is in the root of the CLI project. You need to copy it from there.

**Comment by caroso1222** [comment]
Yeah, definitely. I'll create a `gulp.task('copy:readme')` to copy over that file. I was wondering, do you think we'll need to copy `LICENSE` as well (if applicable)? I've seen it in many node packages within node_modules.

**Comment by NetanelBasal** [comment]
Yes, it's nice to have.
